Sources/Danger/GitLabDSL.swift

```diff
@@ -29,13 +29,52 @@ extension GitLab {
         }
 
         public struct Milestone: Decodable, Equatable {
+            public enum ParentIdentifier: Equatable {
+                case group(Int)
+                case project(Int)
+
+                // MARK: Local error enum
+
+                enum Error: LocalizedError {
+                    case validKeyNotFound
+
+                    // MARK: LocalizedError
+
+                    var errorDescription: String? {
+                        "Not able to find `group_id` or `project_id` from the raw JSON"
+                    }
+                }
+
+                // MARK: Helpers
+
+                public var id: Int {
+                    switch self {
+                    case let .group(id), let .project(id):
+                        return id
+                    }
+                }
+
+                public var isGroup: Bool {
+                    if case .group = self {
+                        return true
+                    } else {
+                        return false
+                    }
+                }
+
+                public var isProject: Bool {
+                    !isGroup
+                }
+            }
+
             public enum CodingKeys: String, CodingKey {
                 case createdAt = "created_at"
                 case description
                 case dueDate = "due_date"
                 case id // swiftlint:disable:this identifier_name
```

Comment:
<!--
  0 failure: 
  1 warning:  SwiftLint rule 'i...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 74;
-->

- :warning: SwiftLint rule 'identifier_name' did not trigger a violation in the disabled region. Please remove the disable command. (`superfluous_disable_command`)